src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor

```diff
@@ -1,28 +1,91 @@
 ﻿@namespace Bit.Client.Web.BlazorUI
 @inherits BitComponentBase
 
-<div @ref="RootElement"
-     style="@StyleBuilder.Value"
-     class="@ClassBuilder.Value">
-    <span>
-        <i class="bit-icon bit-icon--@MessageBarIconName"></i>
-    </span>
-    <span class="@((Truncated && !ElementTruncateState) || IsMultiline ? "not-truncate" : "")">
-        @ChildContent
-    </span>
-    <span>
-        @if (Truncated)
-        {
-            <span @onclick="ToggleElementTruncate">
-                <i class="bit-icon bit-icon--@(ElementTruncateState ? "DoubleChevronDown" : "DoubleChevronUp" )"></i>
-            </span>
-        }
-
-        @if (OnDismiss.HasDelegate)
-        {
-            <span @onclick="Dismiss">
-                <i class="bit-icon bit-icon--@DismissIconName"></i>
-            </span>
-        }
-    </span>
-</div>
+@if (IsMultiline)
+{
+    <div class=@($"{RootElementClass}--root")
+         style=@StyleBuilder.Value>
+        <div class=@($"{ClassBuilder.Value} {RootElementClass}-multiline {GetTypeCss()}")>
+            <div class=@($"{RootElementClass}-content")>
+
+                <div class=@($"{RootElementClass}-icon") aria-hidden="true">
+                    <i class=@($"bit-icon bit-icon--{MessageBarIcon.IconMap[MessageBarType]}")></i>
```

Comment:
consumers must be able to change this icon as they see fit but right now it changes only by MessageBarType